core/application/app_configuration.hpp

```diff
@@ -145,6 +145,11 @@ namespace kagome::application {
      */
     virtual uint32_t maxWsConnections() const = 0;
 
+    /**
+     * @return Kademlia random walk interval
+     */
+    virtual uint32_t getRandomWalkInterval() const = 0;
```

Comment:
Maybe typed (like `boost::filesystem::path` in this file`
```suggestion
    virtual std::chrono::seconds getRandomWalkInterval() const = 0;
```

Reply:
Fixed